apps/automation-v2/src/tlc/batch-liquidate.ts

```diff
@@ -83,79 +84,85 @@ export async function batchLiquidate(
-          log,
-          tlc,
-          tlc.filters.Liquidated()
-        );
+      // if no liquidation happened, success silently
+      if (events.length === 0) return taskSuccessSilent();
 
-        // check if this is the liquidated event
-        if (liquidatedEv) {
-          events.push({
-            what: 'Liquidated',
-            details: [
-              `account = \`${liquidatedEv.account}\``,
-              `collateralValue = \`${formatBigNumber(
-                liquidatedEv.collateralValue,
-                18,
-                4
-              )}\``,
-              `collateralSeized = \`${formatBigNumber(
-                liquidatedEv.collateralSeized,
-                18,
-                4
-              )}\``,
-              `daiDebtWiped = \`${formatBigNumber(
-                liquidatedEv.daiDebtWiped,
-                18,
-                4
-              )}\``,
-            ],
-          });
-        }
-      }
-    });
+      const txUrl = config.CHAIN.transactionUrl(txReceipt.hash);
+      const metadata: TempleTaskDiscordMetadata = {
+        title: 'TLC Batch Liquidate',
+        events,
+        submittedAt,
+        txReceipt,
+        txUrl,
+      };
 
-    // if no liquidation happened, success silently
-    if (events.length === 0) return taskSuccessSilent();
-
-    const txUrl = config.CHAIN.transactionUrl(txReceipt.hash);
-    const metadata: TempleTaskDiscordMetadata = {
-      title: 'TLC Batch Liquidate',
-      events,
-      submittedAt,
-      txReceipt,
-      txUrl,
-    };
-
-    // Send notification
-    const message = await buildTempleTasksDiscordMessage(
-      provider,
-      config.CHAIN,
-      metadata
-    );
-    await discord.postMessage(message);
-
-    // Send discord alert if signer wallet doesn't have sufficient eth balance
-    const ethBalance = await provider.getBalance(walletAddress);
-    if (ethBalance < config.MIN_ETH_BALANCE) {
-      const ethBalanceMessage = await buildDiscordMessageCheckEth(
+      // Send notification
+      const message = await buildTempleTasksDiscordMessage(
+        provider,
         config.CHAIN,
-        submittedAt,
-        walletAddress,
-        ethBalance,
-        config.MIN_ETH_BALANCE
+        metadata
       );
+      await discord.postMessage(message);
+
+      // Send discord alert if signer wallet doesn't have sufficient eth balance
+      const ethBalance = await provider.getBalance(walletAddress);
+      if (ethBalance < config.MIN_ETH_BALANCE) {
+        const ethBalanceMessage = await buildDiscordMessageCheckEth(
+          config.CHAIN,
+          submittedAt,
+          walletAddress,
+          ethBalance,
+          config.MIN_ETH_BALANCE
+        );
 
-      await discord.postMessage(ethBalanceMessage);
+        await discord.postMessage(ethBalanceMessage);
+      }
+    } catch (e) {
+      if (isError(e, 'INSUFFICIENT_FUNDS')) {
+        return taskFailOnce(e.message);
+      }
```

Comment:
is this the correct way to handle & fail once this specific error @mountainpath9 @frontier159 ?